test/shopify_app/managers/webhooks_manager_test.rb

```diff
@@ -22,9 +22,10 @@ class ShopifyApp::WebhooksManagerTest < ActiveSupport::TestCase
       path: "webhooks/orders_updated",
       handler: OrdersUpdatedJob,
       fields: nil,
+      metafield_namespaces: nil,
     }
 
-    ShopifyAPI::Webhooks::Registry.expects(:add_registration).with(expected_hash).once
+    ShopifyAPI::Webhooks::Registry.expects(:add_registration).with(**expected_hash).once
```

Comment:
does this ** change come about from the rest of the diff here, or is it just cleanup? (just for my ruby understanding)